src/js/utils/ErrorMessageUtil.js

```diff
@@ -3,6 +3,36 @@ const MESSAGE_VARIABLE = /\{\{([^{}]+)\}\}/g;
 const ErrorMessageUtil = {
 
   /**
+   * This function returns the `message` of the given array object,
+   * with the path and message components concatenated into a uniform
+   * string that can be displayed unanchored.
+   *
+   * @param {Object} error - The error message
+   * @param {Array} pathTranslationRules - The path translation rules
+   * @returns {String} Returns the composed error string
+   */
+  getUnanchoredErrorMessage(error, pathTranslationRules) {
```

Comment:
Discussed the topic with @wavesoft offline. We're going to keep it as is until there is a use-case where we need translate/concat separately.